assets/js/googlesitekit/notifications/components/common/ActionsCTALinkDismiss.js

```diff
@@ -29,21 +29,26 @@ import CTALink from './CTALink';
 
 export default function ActionsCTALinkDismiss( {
 	id,
+	className = 'googlesitekit-publisher-win__actions',
 	ctaLink,
 	ctaLabel,
+	onCTAClick,
 	dismissLabel = __( 'OK, Got it!', 'google-site-kit' ),
 	dismissExpires = 0,
 } ) {
-	const isNavigatingToCTALink = useSelect( ( select ) =>
-		select( CORE_LOCATION ).isNavigatingTo( ctaLink )
-	);
+	const isNavigatingToCTALink = useSelect( ( select ) => {
+		return ctaLink
+			? select( CORE_LOCATION ).isNavigatingTo( ctaLink )
+			: false;
+	} );
 
 	return (
-		<div className="googlesitekit-publisher-win__actions">
+		<div className={ className }>
```

Comment:
We shouldn't allow developers to _replace_ the existing class name, I'd think… so we should do this instead:

```suggestion
		<div className={ classnames( 'googlesitekit-publisher-win__actions', className ) }>
```

Reply:
I understand, but here, the class name is essentially a flag for the version we need to use for this component. Unfortunately, `googlesitekit-publisher-win__actions` and `googlesitekit-setup-cta-banner__actions-wrapper` are conflicting with each other - so we can either apply one or the other. If we were to use `classnames`, we would have to create another flag like `type` or something solely for styling. Type could then be `banner-notification` or `setup-cta`. And then we'd have to switch the classname based on the type. Ideally, we should be refactoring the legacy banner notifications (viz. all classes with `googlesitekit-publisher-win*`) soon.

So is this worth add the whole type flag, etc or should I leave this as is?